### PR TITLE
African updates

### LINKED ISSUE #12595: Add: Dabanga TV [SD]
### Channel ID

DabangaTV.sd

### Stream URL (optional)

https://cdn-cf.dabangasudan.org/hls/stream.m3u8

### Notes (optional)

1080p

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #12596: Add: Sudania 24 TV [SD]
### Channel ID

Sudania24TV.sd

### Stream URL (optional)

https://samson.streamerr.co:8081/sudania24/index.m3u8

### Notes (optional)

1080p

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by Carlinhos027**
Closes #12595
Closes #12596

### CODE REVIEW

**Comment by Carlinhos027** [comment]
I think it's done for now

**Review by BellezaEmporium — DISMISSED** [review]

File: streams/ci.m3u [hunk 1]
```diff
@@ -9,7 +9,15 @@ https://nci-live.secure2.footprint.net/nci/nci.isml/.m3u8
 https://strhlslb01.streamakaci.tv/str_ntv_ntv/str_ntv_ntv_multi/playlist.m3u8
 #EXTINF:-1 tvg-id="RefletTV.ci",REFLET TV (1080p)
 https://edge-a3.evrideo.tv/8f37c9f0-fe22-44f4-b64a-76ad11730daf_1000026630_HLS/manifest.m3u8
-#EXTINF:-1 tvg-id="RTI1.ci",RTI 1
+#EXTINF:-1 tvg-id="RTI1.ci" status="online",RTI 1 (1080p)
```
Comment (BellezaEmporium) [inline streams/ci.m3u:12]:
```suggestion
#EXTINF:-1 tvg-id="RTI1.ci",RTI 1 (1080p)
```

File: streams/ci.m3u [hunk 2]
```diff
@@ -9,7 +9,15 @@ https://nci-live.secure2.footprint.net/nci/nci.isml/.m3u8
 https://strhlslb01.streamakaci.tv/str_ntv_ntv/str_ntv_ntv_multi/playlist.m3u8
 #EXTINF:-1 tvg-id="RefletTV.ci",REFLET TV (1080p)
 https://edge-a3.evrideo.tv/8f37c9f0-fe22-44f4-b64a-76ad11730daf_1000026630_HLS/manifest.m3u8
-#EXTINF:-1 tvg-id="RTI1.ci",RTI 1
+#EXTINF:-1 tvg-id="RTI1.ci",RTI 1 (1080p)
 https://www.enovativecdn.com/rticdn/smil:rti1.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="RTI2.ci",RTI 2
+#EXTINF:-1 tvg-id="RTI2.ci" status="online",RTI 2 (1080p)
```
Comment (BellezaEmporium) [inline streams/ci.m3u:14]:
```suggestion
#EXTINF:-1 tvg-id="RTI2.ci",RTI 2 (1080p)
```

File: streams/ke.m3u [hunk 3]
```diff
@@ -1,9 +1,17 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="InooroTV.ke",Inooro Tv (720p) [Geo-blocked]
+#EXTINF:-1 tvg-id="InooroTV.ke" status="online",Inooro TV (720p)
```
Comment (BellezaEmporium) [inline streams/ke.m3u:2]:
```suggestion
#EXTINF:-1 tvg-id="InooroTV.ke",Inooro TV (720p)
```

File: streams/mr.m3u [hunk 4]
```diff
@@ -1,3 +1,3 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="",Télévision de Mauritanie (1080p)
+#EXTINF:-1 tvg-id="Elmouritania.mr" status="online",Télévision de Mauritanie (1080p) [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/mr.m3u:2]:
```suggestion
#EXTINF:-1 tvg-id="Elmouritania.mr",Télévision de Mauritanie (1080p) [Not 24/7]
```

File: streams/ng.m3u [hunk 5]
```diff
@@ -40,7 +42,7 @@ https://cdn.odysee.live/content/0edfb4b7fb52d2d5ae30e052ce6b61d376fcd662/master.
 https://media2.streambrothers.com:19360/8038/8038.m3u8
 #EXTINF:-1 tvg-id="TVC.ng",TVC (1080p)
 https://tvce.gridpapaservers.com/TVCSEPT/ngrp:myStream_all/playlist.m3u8
-#EXTINF:-1 tvg-id="TVCNews.ng",TVC News
+#EXTINF:-1 tvg-id="TVCNews.ng" status="online",TVC News (1080p) [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/ng.m3u:45]:
```suggestion
#EXTINF:-1 tvg-id="TVCNews.ng",TVC News (1080p) [Not 24/7]
```

File: streams/sd.m3u [hunk 6]
```diff
@@ -1,9 +1,13 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AfricaTV1.ng",Africa TV 1
+#EXTINF:-1 tvg-id="AfricaTV1.ng",Africa TV 1 (720p)
 http://africatv.live.net.sa:1935/live/africatv/playlist.m3u8
-#EXTINF:-1 tvg-id="AfricaTV2.ng",Africa TV 2
+#EXTINF:-1 tvg-id="AfricaTV2.ng" status="online",Africa TV 2 (720p)
```
Comment (BellezaEmporium) [inline streams/sd.m3u:4]:
```suggestion
#EXTINF:-1 tvg-id="AfricaTV2.ng",Africa TV 2 (720p)
```

File: streams/ug.m3u [hunk 7]
```diff
@@ -1,19 +1,21 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="ArkTV.ug",Ark TV (576p)
 https://arktelevision.org/hlslive/test/test.m3u8
-#EXTINF:-1 tvg-id="BukeddeTV2.ug",Bukedde TV 2 (576p)
-https://stream.hydeug.com/bukedde2flussonic/index.m3u8
-#EXTINF:-1 tvg-id="DreamTV.ug",Dream TV
-http://128.199.114.2:8080/hls/stream.m3u8
-#EXTINF:-1 tvg-id="NTVUganda.ug",NTV Uganda [Not 24/7]
+#EXTINF:-1 tvg-id="BukeddeTV1.ug" status="online",Bukedde TV 1 (576p)
```
Comment (BellezaEmporium) [inline streams/ug.m3u:4]:
```suggestion
#EXTINF:-1 tvg-id="BukeddeTV1.ug",Bukedde TV 1 (576p)
```

File: streams/ug.m3u [hunk 8]
```diff
@@ -1,19 +1,21 @@
...
+https://stream.hydeinnovations.com/bukedde1flussonic/index.m3u8
+#EXTINF:-1 tvg-id="BukeddeTV2.ug" status="online",Bukedde TV 2 (576p) [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/ug.m3u:6]:
```suggestion
#EXTINF:-1 tvg-id="BukeddeTV2.ug",Bukedde TV 2 (576p) [Not 24/7]
```

File: streams/ug.m3u [hunk 9]
```diff
@@ -1,19 +1,21 @@
...
+https://stream.hydeinnovations.com/bukedde2flussonic/index.m3u8
+#EXTINF:-1 tvg-id="NTVUganda.ug" status="online",NTV Uganda [Not 24/7]
```
Comment (BellezaEmporium) [inline streams/ug.m3u:8]:
```suggestion
#EXTINF:-1 tvg-id="NTVUganda.ug",NTV Uganda [Not 24/7]
```

File: streams/ug.m3u [hunk 10]
```diff
@@ -1,19 +1,21 @@
...
 https://ythls.onrender.com/channel/UCwga1dPCqBddbtq5KYRii2g.m3u8
-#EXTINF:-1 tvg-id="SaltTV.ug",Salt TV
+#EXTINF:-1 tvg-id="SaltTV.ug" status="online",Salt TV (720p)
```
Comment (BellezaEmporium) [inline streams/ug.m3u:10]:
```suggestion
#EXTINF:-1 tvg-id="SaltTV.ug",Salt TV (720p)
```

File: streams/ug.m3u [hunk 11]
```diff
@@ -1,19 +1,21 @@
...
 #EXTINF:-1 tvg-id="TVWest.ug",TV West (720p)
 https://stream.hydeug.com/tvwest-flussonic/index.m3u8
 #EXTINF:-1 tvg-id="UrbanTV.ug",Urban TV (576p)
 https://stream.hydeug.com/urban-flussonic/index.m3u8
 #EXTINF:-1 tvg-id="WanLuoTV.ug",Wan Luo TV (576p)
 https://stream.hydeug.com/luotv-flussonic/index.m3u8
-#EXTINF:-1 tvg-id="",Fort TV (480p)
+#EXTINF:-1 tvg-id="DreamTV.ug",Dream TV (480p)
+http://128.199.114.2:8080/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" status="online",Fort TV (480p)
```
Comment (BellezaEmporium) [inline streams/ug.m3u:20]:
```suggestion
#EXTINF:-1 tvg-id="",Fort TV (480p)
```

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

File: streams/ci.m3u [hunk 12]
```diff
@@ -9,7 +9,15 @@ https://nci-live.secure2.footprint.net/nci/nci.isml/.m3u8
...
+#EXTINF:-1 tvg-id="RTI2.ci",RTI 2 (1080p)
 https://www.enovativecdn.com/rticdn/smil:rti2.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="BenieTV.ci",Benie TV (720p)
+https://voozmedia.fun/benietv/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="ChristLive.ci",Christ Live (720p)
+https://voozmedia.fun/stream/christlive/playlist.m3u8
+#EXTINF:-1 tvg-id="MiracleTVPlus.ci",Miracle TV+ (720p)
+https://strhlslb01.streamakaci.tv/str_mtv_mtv/str_mtv_multi/playlist.m3u8
+#EXTINF:-1 tvg-id="RTILa3.ci",RTI La 3 (1080p) [Not 24/7]
+https://www.enovativecdn.com/rticdn/smil:rti3.smil/playlist.m3u8
```
Comment (trincowski) [inline streams/ci.m3u:23]: > https://www.enovativecdn.com/rticdn/smil:rti3.smil/playlist.m3u8

I have never saw this link working. Have you ever seen it online?

Comment (BellezaEmporium) [reply streams/ci.m3u:23]: Geo, for Ivoirian people only.

**Pull request merged**